report_generation/models/data_extractor.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+
+from models.parcel_perfect_database_client import ParcelPerfectDatabaseClient
+from utils.log_execution_time_decorator import log_execution_time
+
+
+class DataExtractor:
+    def __init__(self, database_data: dict) -> None:
+        self.database_data = database_data
+        self.pp_client = ParcelPerfectDatabaseClient(**database_data)
+
+    def _waybill_analysis_view(self) -> str:
```

Comment:
@TashiSoldin, I need you to go through the data they are sending us and we need to ensure we have all the right columns (in order ideally). They have slightly different names but I have most. I cc'd you in an email with the mapping. 

I will get the right date range from the ladies